Echo.json5

```diff
@@ -30,6 +30,7 @@
     { name: 'AROptionsNewInsightsPage', value: false },
     { name: 'AREnableReactNativeWebView', value: true },
     { name: 'AROptionsNewArtistInsightsPage', value: true },
+    { name: 'AROptionsInquiryCheckout', value: true },
```

Comment:
Wonder if the presence of the key matter for Eigen. Do I need to set this flag to `false`, deploy it to S3 so that Eigen can [refer to it](https://github.com/artsy/eigen/pull/4782), and then update it to `true` when we are ready to release? Or is it ok to not have the Echo key and simply deploy this when we are ready?

Reply:
Some more thoughts here: https://github.com/artsy/eigen/pull/4782#issuecomment-841946516

Reply:
You don't need to add it on echo ahead of time. **But**, if the flag is missing from echo, and eigen has an `echoFlagKey`, then it will assume it's `true`. The proposed order would be to add the flag in eigen, make it work, then add it on echo, then turn the `readyForRelease` to `true`, which will make eigen take the echo value.